src/neural/onnx/builder.cc

```diff
@@ -289,7 +290,9 @@ std::string OnnxBuilder::Identity(const std::string& name,
 std::string OnnxBuilder::Selu(const std::string& name,
                               const std::string& input) {
   auto* node = model_.mutable_graph()->add_node();
-  return PopulateStdNodeFields(node, name, input, "Selu");
+  auto out = PopulateStdNodeFields(node, name + "/elu", input, "Elu");
+  AddFloatAttribute(node, "alpha", 1.0507);
+  return Mul(name, out, FloatOnnxConst({1.67326}, {1}));
 }
```

Comment:
I would suggest using a flag, but probably doesn't make a difference. Better keep the `Selu` operator unchanged and add an `Elu` one, then make the change in the converter (with a comment).